style.css

```diff
@@ -303,3 +303,45 @@ body {
   text-shadow: 2px;
   width: 20px;
 }
+
+.form {
+  background-color: #6070fffb;
+  border-top-left-radius: 20%;
+  padding: 10%;
+}
+
+.form h1 {
+  text-align: center;
+  color: #fff;
+  font-size: 2rem;
+  text-decoration: none !important;
+}
```

Comment:
Semantic HTML does not Incurage to use important so I recommend using sematic HTML and CSS practices